src/tools/tests/schema2kotlin-test.ts

```diff
@@ -0,0 +1,46 @@
+/**
+ * @license
+ * Copyright (c) 2020 Google Inc. All rights reserved.
+ * This code may only be used under the BSD style license found at
+ * http://polymer.github.io/LICENSE.txt
+ * Code distributed by Google as part of this project is also
+ * subject to an additional IP rights grant found at
+ * http://polymer.github.io/PATENTS.txt
+ */
+
+
+import {assert} from '../../platform/chai-node.js';
+import {KotlinGenerator} from '../schema2kotlin.js';
+import {SchemaNode} from '../schema2graph.js';
+import {Schema} from '../../runtime/schema.js';
+
+
+describe('schema2wasm', () => {
+  describe('kotlin-generator', () => {
```

Comment:
`describe('mapOf')`?

Reply:
Will break out these tests into the kt gen utils in a PR today.

Reply:
Not done?

I just meant: change the name of this suite of tests. It's testing the `mapOf` function only, it's not testing other stuff.